projects/duplicate_crash_pings.kp/knowledge.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Duplicate Crash Pings
+authors:
+- chutten
+tags:
+- duplicate
+- dedupe
+- crash
+created_at: 2017-04-07 00:00:00
+updated_at: 2017-04-07 13:38:11.063912
+tldr: When the patches landed to dedupe crash pings (bug 1354468 has the list), did
+  they work?
+thumbnail: images/output_11_0.png
+---
+### How many duplicate crash pings are we receiving on Nightly/Aurora from 2017-03-28 - 2017-04-07?
```

Comment:
Oh right, that was before I opened up the analysis to show the entire "duped ping life-cycle" :S